docker-compose.yaml

```diff
@@ -21,6 +21,7 @@ services:
       - "4005:4005"
     depends_on:
       - redis
+      - pisp-simulator
```

Comment:
why do we need here `pisp-simulator` (it is specified in `docker/docker-compose.yaml` for integration testing purposes) and this root `docker-compose.yaml` is only dedicated to starting Inbound, Outbound, and Redis only -> the raw setup without any simulators

Reply:
well I used it because internally it is calling /signchallenge in simulator and I didn't find it under thirdparty-scheme-adapter APIs